src/shader/shaderSheenFunctions.js

```diff
@@ -0,0 +1,97 @@
+export const shaderSheenFunctions = /* glsl */`
+
+float velvetD( float cosThetaH, float roughness ) {
+
+    float alpha = max( roughness, 0.07 );
+    alpha = alpha * alpha;
+
+    float invAlpha = 1.0 / alpha;
+
+    float sqrCosThetaH = cosThetaH * cosThetaH;
+    float sinThetaH = max( 1.0 - sqrCosThetaH, 0.001 );
+
+    return ( 2.0 + invAlpha ) * pow( sinThetaH, 0.5 * invAlpha ) / ( 2.0 * PI );
+
+}
+
+float velvetParamsInterpolate( int i, float oneMinusAlphaSquared ) {
+
+    const float p0[5] = float[5]( 25.3245, 3.32435, 0.16801, -1.27393, -4.85967 );
+    const float p1[5] = float[5]( 21.5473, 3.82987, 0.19823, -1.97760, -4.32054 );
+
+    return mix( p1[i], p0[i], oneMinusAlphaSquared );
+
+}
+
+float velvetL( float x, float alpha ) {
```

Comment:
Are there references for all these other sheen functions, as well, that we could add a comment for?

Reply:
In `KHR_materials_sheen` the function `l` is implemented here with `velvetL` and `velvetParamsInterpolate` and `sheenDistribution` from `KHR_materials_sheen` is implemented in `velvetD`.

Reply:
I'll search through the references on `KHR_materials_sheen` and annotate the functions with them with links / page number etc.